benchmark/fs/bench-copyFileSync.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+const common = require('../common');
+const fs = require('fs');
+const tmpdir = require('../../test/common/tmpdir');
+tmpdir.refresh();
+
+const bench = common.createBenchmark(main, {
+  type: ['invalid', 'valid'],
+  n: [1e4],
```

Comment:
Why `1e4` instead of `1e5`?

Reply:
It takes a lot of time to just run the benchmarks with the suggested value.